features/step_definitions/basic_steps.rb

```diff
@@ -12,6 +12,39 @@
     visit new_user_registration_path
 end
 
+When('I fill in {string} field with {string}') do |field, input|
+    fill_in field, with: input
+end
+
 When('I click {string}') do |element|
     click_on element
-end
\ No newline at end of file
+end
+
+When('I am on the login page') do
+    visit new_user_session_path
+end
+
+When('I press {string}') do |link_name|
+    click_link link_name
+end
+
+When('I am on the {string} page') do |page_name|
+    visit root_path
+end
+
+Given('I fill in {string} with {string}') do |field, email|
+    fill_in field, with: email
+end
+
+=begin
+Given('I tick remember me') do
+    check 'user_remember_me'
+end
+=end
```

Comment:
comments in code = Shame

Reply:
Ha... We commented them right before review.  Fixed.